app/maps/mapSelection.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+import _ from 'lodash';
+
+function _getMapListFavouringUpvotedMaps(maps, keyword) {
+    let biasedMapList = [];
```

Comment:
I've looked it up, apparently you can spell it 'biassed' (double s), but I don't think people under the age of 120 use that spelling anymore ;D